src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2797,9 +2797,9 @@ def test_vmss_run_command_e2e(self, resource_group, resource_group_location):
         self.cmd('vmss run-command list -l {loc}')
         self.cmd('vmss run-command show --command-id RunShellScript -l {loc}')
 
-        self.cmd('vmss create -g {rg} -n {vmss} --image ubuntults --admin-username clitest1 --generate-ssh-keys').get_output_in_json()
+        self.cmd('vmss create -g {rg} -n {vmss} --image ubuntults --admin-username clitest1 --instance-count 1 --generate-ssh-keys').get_output_in_json()
```

Comment:
please also add `--disable-overprovision`, so make sure we only have 1 instance all the time.

Reply:
@yugangw-msft sounds good!